WordPressKitTests/WordPressComOAuthClientTests.swift

```diff
@@ -277,10 +271,11 @@ class WordPressComOAuthClientTests: XCTestCase {
                 XCTFail("Expected to fail, but run the success block")
             },
             failure: { error in
-                expect.fulfill()
-                XCTAssertEqual(error.domain, WordPressComOAuthClient.WordPressComOAuthErrorDomain)
-                XCTAssertEqual(error.code, WordPressComOAuthError.unknown.rawValue)
-                XCTAssertEqual(error.localizedDescription, "Response requires handling the MFA Webauthn flow but handler given ('needsMultifactor' parameter).")
+                if case .missingMultiFactorHandler = error {
+                    expect.fulfill()
+                } else {
+                    XCTFail("Unexpected error: \(error)")
+                }
```

Comment:
All other unit test fixes are adopting the new signature. This one here is the only place where assertion has been changed. Because the `missingMultiFactorHandler` error should never happen in production, it's a programming error and it's not really helpful to show the "require 'needsMultifactor' parameter" message to users.

Reply:
> Because the missingMultiFactorHandler error should never happen in production, it's a programming error and it's not really helpful to show the "require 'needsMultifactor' parameter" message to users.

This reminds me, I think the multifactor handler is optional because the intent was to not break compatibility, but given this PR already introduces a breaking change, then everything is fair game. I would offer to make the change myself, but I'm doing support rotation this week. That's not to say I won't be able to do that next week, but just in case you are keen and want to jump ahead...

Reply:
It's removed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/650/commits/c1411cd0e7471aa34b15b2e09d942d15d38965d6